omod/src/test/java/org/openmrs/module/idgen/web/controller/AutoGenerationOptionControllerTest.java

```diff
@@ -0,0 +1,152 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.idgen.web.controller;
+
+import org.apache.commons.beanutils.PropertyUtils;
+import org.apache.poi.hssf.record.formula.functions.True;
+import org.codehaus.jackson.map.ObjectMapper;
+import org.junit.Before;
+import org.junit.Test;
+import org.openmrs.PatientIdentifierType;
+import org.openmrs.module.idgen.AutoGenerationOption;
+import org.openmrs.module.idgen.service.IdentifierSourceService;
+import org.openmrs.api.context.Context;
+import org.openmrs.module.webservices.rest.SimpleObject;
+import org.openmrs.module.webservices.rest.test.Util;
+import org.openmrs.module.webservices.rest.web.RestConstants;
+import org.openmrs.module.idgen.web.RestTestConstants;
+import org.openmrs.module.webservices.rest.web.v1_0.controller.MainResourceControllerTest;
+import org.springframework.mock.web.MockHttpServletRequest;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+
+
+/**
+ * Tests CRUD operations for {@link AutoGenerationOption}s via web service calls
+ */
+public class AutoGenerationOptionControllerTest extends MainResourceControllerTest {
+  
+  private IdentifierSourceService service;
+
+  public PatientIdentifierType ptid;
+
+  @Before
+	public void before() throws Exception {
```

Comment:
Same indention issue for the contents of this class. Can you look at the indention issues in all files?

Reply:
The indentation looks normal on my machine here
<img width="1278" alt="screen shot 2017-10-05 at 4 53 22 pm" src="https://user-images.githubusercontent.com/28529978/31237043-bc556fee-a9ed-11e7-98f3-ec6b1de69cb2.png">

Reply:
@andela-dbamidele  check this out https://wiki.openmrs.org/display/docs/Developer+How-To+Setup+And+Use+Eclipse .  

and start using eclipse